src/sql/util.py

```diff
@@ -494,3 +497,46 @@ def get_default_configs(sql):
     del default_configs["parent"]
     del default_configs["config"]
     return default_configs
+
+
+def get_connections_config_file():
```

Comment:
do we need all these functions? the connection widget is already loading from the dsn_file (configured via `%config Sqlmagic.dsn_file`, so I don't understand why we need all of this parsing logic.

for this PR, we only need to show the widget. that's it.

am I missing anything?

Reply:
The current process is 
1. running `%load_ext sql` loads connections from `connections.ini` file 
2. running `%sqlcmd connect` displays the widget that shows loaded connections from step 1

`get_connections_config_file` is used in `load_conn_config`, and `load_conn_config` is used in step 1 from above to load connections.

> 2. When there is existing connection in a connections.ini file, ensure it can be connected and displayed through ConnectorWidget

It seems I think I misunderstood the second point from the [AC](https://github.com/ploomber/jupysql/issues/632#issuecomment-1648646267) and added step 1.

Reply:
sounds good!

Reply:
After deleting the functions from util.py, there is no change in the util.py compared to the master branch, so I can't add a link.

Now this PR is only about displaying the widget when a user runs `%sqlcmd connect` and testing whether the widget is displayed no matter a user sets the connection.ini file or not.

Reply:
Aside from that, is there a specific reason why the `stored_connections` from the `ConnectorWidget` is a class variable? [code here](https://github.com/ploomber/jupysql-plugin/blob/main/jupysql_plugin/widgets/connector_widget.py#L253)

I tried adding an assert statement for checking the `stored_connections` to confirm the correct information is loaded, but because the `stored_connections` is a class variable, it uses connections.ini from my local jupysql package folder when we import `ConnectorWidget` from [connect.py](https://github.com/ploomber/jupysql/pull/783/files#diff-97c7661adb8698813af7c891197f6961ea293d1d1b02be2d5928d07db813bbe2R1-R4).

But when I run a notebook in a completely different folder, it works as expected. (Use the connections.ini file in that different folder).

Reply:
I didn't write the connectionwidget, so I can't comment on that. I'm actually working on some fixes so ensure that all of the things you found are documented in a in issue in the jupysql-plugin repository, so I can tackle them.